arch/arm/core/aarch64/Kconfig

```diff
@@ -58,6 +58,12 @@ config IPM_CONSOLE_STACK_SIZE
 
 if CPU_CORTEX_A
 
+config ARMV8_A_NS
+	bool "ARMv8-A Normal World (Non-Secure world of Trustzone)"
+	help
+	  This option signifies the use of an ARMv8-A processor
```

Comment:
Wouldn't be better to say that is Zephyr that is running in Trustzone Normal World?

Reply:
Done